src/Features/authentication.feature

```diff
@@ -34,45 +34,45 @@ Feature: When an user needs to authenticate
     Then I should see "urn:oasis:names:tc:SAML:2.0:status:Success"
 
     And the logs are:
-      | level  | message                                                                                                                         | sari    |
+      | level  | message                                                                                                                                 | sari    |
```

Comment:
The majority of the changed lines are only reformatting the file. This obfuscates the actual change. Can you revert the styling? Or make it a 2 commit business?

Reply:
This has now been split up into two commits